include/swift/Driver/DependencyGraph.h

```diff
@@ -228,9 +230,10 @@ class DependencyGraph : public DependencyGraphImpl {
   /// ("depends") are not cleared; new dependencies are considered additive.
   ///
   /// If \p node has already been marked, only its outgoing edges are updated.
-  LoadResult loadFromPath(T node, StringRef path) {
-    return DependencyGraphImpl::loadFromPath(Traits::getAsVoidPointer(node),
-                                             path);
+  LoadResult loadFromPath(T node, StringRef path,
+                          const bool EnableExperimentalDependencies) {
```

Comment:
Are you even going to use the same dependency graph structure? It might be better to not bother with this flag at all.

(Then again, you can always take it back out.)

Reply:
Yes, you are right! Taking it out.